taskfiles/utils.yml

```diff
@@ -50,7 +50,7 @@ tasks:
         vars:
           DATA_DIR: "{{.DATA_DIR}}"
           EXCLUDE_PATHS:
-            ref: ".EXCLUDE_PATHS"
+            ref: "default (list) .EXCLUDE_PATHS"
```

Comment:
I looked at https://taskfile.dev/reference/templating/#default-functions and believe our usage matches the explanations there.